cdk/domino_cdk/agent.py

```diff
@@ -111,16 +111,70 @@ def generate_install_config(
                     "service.beta.kubernetes.io/aws-load-balancer-backend-protocol": "tcp",
                     "service.beta.kubernetes.io/aws-load-balancer-ssl-ports": "443",
                     "service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout": "3600",  # noqa
-                    "service.beta.kubernetes.io/aws-load-balancer-proxy-protocol": "*",
                     # "service.beta.kubernetes.io/aws-load-balancer-security-groups":
                     #     "could-propagate-this-instead-of-create"
                 },
-                "targetPorts": {"http": "http", "https": "http"},
                 "loadBalancerSourceRanges": install.access_list,
             },
         }
     }
 
+    if install.istio_compatible:
+        agent_cfg = DominoCdkUtil.deep_merge(
+            agent_cfg,
+            {
+                "istio": {
+                    "enabled": True,
+                    "install": True,
+                    "cni": False,
+                },
+                "services": {
+                    "nginx_ingress": {
+                        "chart_values": {
+                            "controller": {
```

Comment:
Wonder if it would be cleaner if you just had `agent_cfg["istio"] = {<istio config>}`, and then did the deep merge on just the contents of controller here?